docs/book_examples/src/env_md.rs

```diff
@@ -2,7 +2,8 @@ use druid::widget::Label;
 use druid::{Color, Key, WidgetExt};
 
 // ANCHOR: key_or_value
-const IMPORTANT_LABEL_COLOR: Key<Color> = Key::new("my-app.important-label-color");
+const IMPORTANT_LABEL_COLOR: Key<Color> =
+    Key::new("org.linebender.druid.theme.my-app.important-label-color");
```

Comment:
This isn't part of druid or the theme, so I'd probably do something like,

```suggestion
    Key::new("org.linebender.example.important-label-color");
```

Reply:
Committing suggestions will probably fail rustfmt, will resolve these manually.